### ISSUE TITLE
Correct Video Reward for KIS

### ISSUE THREAD

**Issue by klschoef** [lucaro/DRES#496]
For KIS tasks it would be good to reward finding the correct video,  because often a video contains similar or even equal scenes (for example in news videos, where there is a preview of a scene). Such an example is given in V3C video 05579, which contains an almost identical scene at 02:52 and 16:23 (happened at IVR4B2024 at Task v7).

<img width="1610" alt="Screenshot 2024-09-19 at 10 38 39" src="https://github.com/user-attachments/assets/20d960ca-0aa9-48eb-8911-8c047d06fbbc">
<img width="1567" alt="Screenshot 2024-09-19 at 10 38 24" src="https://github.com/user-attachments/assets/b8401d66-904f-4c3d-9156-34f385941a09">

One possibility would be to give half the points for finding a scene in the correct video (e.g., 500 - <time-decrease>; instead of 1000 - <time-decrease>).

**Comment by lucaro** [comment]
Adding further scoring functions is not a problem. What would you propose the actual function to look like @klschoef ?

**Comment by klschoef** [comment]
As far as I know the KIS scoring currently is a follows:
int(max(0, r^i(t) - p^i(ws) ))
r^i(t) = (1000 - s_{min}) * ((t_{max} - t) / t_{max} + s_{min}) 
p^i(ws) = ws * 100

We could modify it for KIS tasks so that for correct videos, r could be used like this (thanks to Omar for this suggestion):
r^i(t) = (500 - s_{min}) * ((t_{max} - t) / t_{max} + s_{min}) 

If the correct segment in the video is found later, the original function could be used (and maybe the previously submitted segment in the same video  could be considered as a wrong submission).  

But I think this should be also discussed with the VBS teams.

**Comment by lucaro** [comment]
This would not consider how far away a submitted solution is from the target. Is that what we want, or would we want to consider that distance?

**Comment by klschoef** [comment]
Yes, I think this is what we want (a reward for finding the correct video).
